kitsune/kpi/tests/test_cron.py

```diff
@@ -161,15 +161,36 @@ class CronJobTests(TestCase):
     def test_update_visitors_cron(self, visitors):
         """Verify the cron job inserts the right rows."""
         visitor_kind = MetricKindFactory(code=VISITORS_METRIC_CODE)
-        visitors.return_value = {"2012-01-13": 42, "2012-01-14": 193, "2012-01-15": 33}
+
+        today = date.today()
```

Comment:
Didn't we experience issues previously with dates in tests when we rely on server time?

Reply:
In this case, what matters is how the following dates -- `days_ago_*` -- relate to `today`, no matter what it ends up being, so I don't think there's any chance of it causing issues.